url.bs

```diff
@@ -2483,6 +2483,8 @@ interface URL {
            attribute USVString search;
   [SameObject] readonly attribute URLSearchParams searchParams;
            attribute USVString hash;
+
+  USVString toJSON();
```

Comment:
Even though I'm not a fan of the complicated serializer syntax, I do like the conciseness of just writing

```idl
serializer;
```

which is more parallel to the existing `iterable<USVString, USVString>;` for `URLSearchParams`.